core/io/resource_uid.cpp

```diff
@@ -29,14 +29,17 @@
 /*************************************************************************/
 
 #include "resource_uid.h"
+#include "core/config/project_settings.h"
 #include "core/crypto/crypto.h"
 #include "core/io/dir_access.h"
 #include "core/io/file_access.h"
```

Comment:
This was missing previously but I like my includes clean :P
```suggestion
#include "resource_uid.h"

#include "core/config/project_settings.h"
#include "core/crypto/crypto.h"
#include "core/io/dir_access.h"
#include "core/io/file_access.h"
```